style.css

```diff
@@ -0,0 +1,406 @@
+body {
+  background: #e5e5e5;
+}
+
+.hex {
+  color: #6070ff;
+}
+
+a:hover {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+@media only screen and (min-width: 375px) and (max-width: 768px) {
```

Comment:
- [ ] Please don't use media query for mobile-first approach you will use media queries for other devices so kindly remove this media query :+1: